README.md

```diff
@@ -1,2 +1,174 @@
-# react-capstone-metrics
-A capstone that depicts information about countries via a RESTful API. Their population, landmarks, area, longitude and latitude and a host of others.
+<div align="center">
+  <h3><b>React Capstone Metrics</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖React Capstone Metrics <a name="about-project"></a>
+
+**A capstone that depicts information about countries via a RESTful API. Their population, landmarks, area, longitude and latitude and a host of others.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>API</summary>
+  <ul>
+    <li><a href="https://restcountries.com/">Country Api</a>a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Show countries information**
+- **Displays continent information**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- [Live Demo](soon)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+```sh
+  sudo apt install node.js
+```
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd React Capstone Metrics
+  git clone https://github.com/Theodoraldo/react-capstone-metrics.git
+```
+-
+
+### Install
+
+Install this project with:
+
+
+```sh
+  cd React Capstone Metrics
+  npm build
+```
+-
+
+### Usage
+
+To run the project, execute the following command:
+
+
+```sh
+  npm start
+```
+-
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm test
+```
```

Comment:
- [ ] One of your tests is failing. Kindly fix that.

  ![image](https://github.com/Theodoraldo/react-capstone-metrics/assets/36542986/6c50d34c-85fd-46c9-ac70-7373ad468f4b)